src/InformaticsGateway/Program.cs

```diff
@@ -87,6 +92,20 @@ internal static void InitializeDatabase(IHost host)
                     configureLogging.AddConfiguration(builderContext.Configuration.GetSection("Logging"));
                     configureLogging.AddFile(o => o.RootPath = AppContext.BaseDirectory);
                 })
+                .UseSerilog((context, services, configuration) => configuration
+                    .ReadFrom.Configuration(context.Configuration)
+                    .ReadFrom.Services(services)
+                    .MinimumLevel.Override("Microsoft", LogEventLevel.Information)
+                    .MinimumLevel.Debug()
+                    .Enrich.FromLogContext()
+                    .Enrich.WithExceptionDetails()
+                    .Enrich.WithProperty("dllversion", Assembly.GetEntryAssembly().GetName().Version)
+                    .Enrich.WithProperty("dllName", Assembly.GetEntryAssembly().GetName().Name)
+                    .WriteTo.File(

```

Comment:
Do we still need this if we it's configured in the appsettings.json file?

Reply:
I think we do, we can always remove it later if not !